modules/server_processes_manager/jsx/server_processes_managerIndex.js

```diff
@@ -0,0 +1,114 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Loader from 'Loader';
+import FilterableDataTable from 'FilterableDataTable';
+
+class ServerProcessesManagerIndex extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: {},
+      error: false,
+      isLoaded: false,
+    };
+
+    this.fetchData = this.fetchData.bind(this);
+    this.formatColumn = this.formatColumn.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData()
+      .then(() => this.setState({isLoaded: true}));
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   *
+   * @return {object}
+   */
+  fetchData() {
+    return fetch(this.props.dataURL, {credentials: 'same-origin'})
+      .then((resp) => resp.json())
+      .then((data) => this.setState({data}))
+      .catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   *
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {object} row - row content indexed by column
+   *
+   * @return {*} a formated table cell for a given column
+   */
+  formatColumn(column, cell, row) {
+    return (<td>{cell}</td>);
+  }
+
+  render() {
+    // If error occurs, return a message.
+    // XXX: Replace this with a UI component for 500 errors.
+    if (this.state.error) {
+      return <h3>An error occured while loading the page.</h3>;
+    }
+
+    // Waiting for async data to load
+    if (!this.state.isLoaded) {
+      return <Loader/>;
+    }
+
+   /**
+    * XXX: Currently, the order of these fields MUST match the order of the
+    * queried columns in _setupVariables() in server_processes_manager.class.inc
+    */
+    const fields = [
+      {label: 'PID', show: true, filter: {
+        name: 'pid',
+        type: 'text',
+      }},
+      {label: 'Type', show: true, filter: {
+        name: 'type',
+        type: 'text',
+      }},
+      {label: 'Stdout File', show: true},
+      {label: 'Stderr File', show: true},
+      {label: 'Exit Code File', show: true},
+      {label: 'Exit Code', show: true},
+      {label: 'UserID', show: true, filter: {
```

Comment:
sorry, would really like to follow the new convention here! UserID should be displayed as 'User ID'.